src/esaml.erl

```diff
@@ -398,9 +398,14 @@ to_xml(#esaml_authnreq{version = V, issue_instant = Time, destination = Dest, is
                       #xmlAttribute{name = 'ProtocolBinding', value = "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST"}],
         content = [
             #xmlElement{name = 'saml:Issuer', content = [#xmlText{value = Issuer}]},
-            #xmlElement{name = 'saml:Subject', content = [
-                #xmlElement{name = 'saml:SubjectConfirmation', attributes = [#xmlAttribute{name = 'Method', value = "urn:oasis:names:tc:SAML:2.0:cm:bearer"}]}
-            ]},
+            %% This should not be included in the request for Azure Entra ID
```

Comment:
so there is no need to add Subject at all for other integrations too ?

Reply:
I think so.
Tested:
Okta (signed request and un-signed request)
OneLogin (signed request and un-signed request)
Azure Entra ID (only un-signed request)